pkg/handlers/app.go

```diff
@@ -281,6 +281,13 @@ func responseAppFromApp(a *apptypes.App) (*types.ResponseApp, error) {
 		return nil, errors.Wrap(err, "failed to check if snapshots is allowed")
 	}
 	allowSnapshots := s && license.Spec.IsSnapshotSupported
+	if allowSnapshots {
+		if util.IsEmbeddedCluster() {
+			allowSnapshots = false // snapshots are not allowed in embedded cluster installations today
```

Comment:
why not just:
```
allowSnapshots := s && license.Spec.IsSnapshotSupported && !util.IsEmbeddedCluster()
```

Reply:
because before I rewrote this to use an env var that if statement included getting the k8s clientset etc

it's not required anymore